hw37Promise/index.js

```diff
@@ -0,0 +1,77 @@
+form.addEventListener("submit", getPost);
```

Comment:
You have approximately the same issues with code as other students

Please, check these comments
https://github.com/Kross55/YakushenkoFrontEndPro29.01.23/pull/10/files

And here is how the final code may look
https://github.com/Kross55/YakushenkoFrontEndPro29.01.23/blob/main/homework/hw37PromiseFinal/main.js